todo_app/app.py

```diff
@@ -115,9 +137,33 @@ def login_callback():
         user_management = UserManagement()
         user = user_management.get_user(user_id)
 
+        app.logger.info({
```

Comment:
Is there a particular reason you've switched between the two formats below?
```python
app.logger.info('message here', extra={'category': Category})
# vs
app.logger.info({'message': 'message here', 'category': Category})
```

Reply:
I started with the first format and then discovered the second one. I thought I had changed them all to be the second format, I must have missed some.